cocos/renderer/backend/Types.h

```diff
@@ -164,8 +164,8 @@ struct SamplerDescriptor
     SamplerFilter magFilter = SamplerFilter::LINEAR;
     SamplerFilter minFilter = SamplerFilter::LINEAR;
     SamplerFilter mipmapFilter = SamplerFilter::LINEAR;
-    SamplerAddressMode sAddressMode = SamplerAddressMode::REPEAT;
-    SamplerAddressMode tAddressMode = SamplerAddressMode::REPEAT;
```

Comment:
It is ok to change default value, but we should do some protection of the usage in Texture.

Reply:
got it.